src/main/java/view5d/AGenericDialog.java

```diff
@@ -25,28 +25,10 @@
 // import java.lang.Number.*;
 package view5d;
 
-import java.awt.Button;
-import java.awt.Component;
-import java.awt.Dialog;
+import java.awt.event.*;
+import java.awt.*;
 // import ij.*;
 // import ij.gui.*;
-import java.awt.FlowLayout;
-import java.awt.Frame;
-import java.awt.GridBagConstraints;
-import java.awt.GridBagLayout;
-import java.awt.Insets;
-import java.awt.Label;
-import java.awt.Panel;
-import java.awt.TextArea;
-import java.awt.TextField;
-import java.awt.event.ActionEvent;
-import java.awt.event.ActionListener;
-import java.awt.event.FocusEvent;
-import java.awt.event.FocusListener;
-import java.awt.event.ItemEvent;
-import java.awt.event.ItemListener;
-import java.awt.event.WindowEvent;
-import java.awt.event.WindowListener;
 
 class AGenericDialog extends Dialog implements WindowListener,ActionListener, FocusListener, ItemListener {  // an easy class for generating user interaction
```

Comment:
our static analysis engine detects that your pull request include change from an explicit import to a wildcard import. According to the best practice (http://www.javadude.com/articles/importondemandisevil.html), import star (import on demand) is a bad practice, wildcard import can sometimes cause name conflicts and ambiguities.